tcms/report/data.py

```diff
@@ -51,18 +54,6 @@ def do_nothing(value):
 class ProductBuildReportData(object):
     """Report data by builds of a Product"""
 
-    def total_runs_count(self, product_id, finished=False):
```

Comment:
In the future avoid moving methods around because it makes it impossible to see which lines have been changed exactly.

Reply:
Ok, will keep in mind for the future.